packages/components/src/popover/index.js

```diff
@@ -390,33 +388,37 @@ const Popover = (
 	// we need to manually update the floating's position as the reference's owner
 	// document scrolls. Also update the frame offset if the view resizes.
 	useLayoutEffect( () => {
-		if ( ownerDocument === document ) {
+		if ( referenceOwnerDocument === document ) {
+			frameOffsetRef.current = undefined;
```

Comment:
Resetting `frameOffsetRef.current` when there's no `iframe`

Reply:
Yes, I guess this is possible if the popover anchor changes from something in an iframe to something outside. Good spot 👍